imaging/ml/ml_codelab/README.md

```diff
@@ -18,33 +18,21 @@ PROJECT_NUMBER=`gcloud projects describe ${PROJECT_ID} | grep projectNumber | se
 COMPUTE_ENGINE_SERVICE_ACCOUNT="${PROJECT_NUMBER}-compute@developer.gserviceaccount.com"
 gcloud projects add-iam-policy-binding ${PROJECT_ID} --member "serviceAccount:${COMPUTE_ENGINE_SERVICE_ACCOUNT}" --role roles/pubsub.admin
 ```
-## Create Datalab instance
+## Create AI Platform Notebooks instance
 
-This following shell command will create a SSH tunnel allowing you to visit the Datalab UI in your web browser. You can run this on a shell in your **local machine** or **[Cloud Shell](https://cloud.google.com/shell/docs/)**.
+Follow the steps listed [here](https://cloud.google.com/ai-platform/notebooks/docs/create-new)
+Create Python 3 instance with deafult n1-standard-4 machine 
```

Comment:
There is no such option from the "New Instance" UI. I think they should be using 'Python"? Or perhaps "Tensorflow 1.x" without GPUs? Not sure which one is better or if it makes a difference. They can just use the default machine type, so I actually spell out "n1-standard-4", just say default. The "Python 3" notebook notebook creation is described in one of the next steps and shouldn't be mentioned here (the VM instance supports both Py2 and Py3),

Reply:
>just say default.

done 
>Or perhaps "Tensorflow 1.x" without GPUs? Not sure which one is better or if it makes a difference. 

"Tensorflow 1.x" without GPUs allocates more cpu and memory so it's cost more, in our cases where is no difference because we don't perform training of model on this machines, so we need simplest type to perform api calls